packages/micropip/test_micropip.py

```diff
@@ -458,3 +458,34 @@ def test_list_loaded_from_js(selenium_standalone_micropip):
         `);
         """
     )
+
+
+def test_install_with_credentials(selenium_standalone_micropip):
+    selenium = selenium_standalone_micropip
+    selenium.run_js(
```

Comment:
I think you should be able to use run_in_pyodide to reduce the nesting
```suggestion
@run_in_pyodide(standalone=True)
def test_install_with_credentials():
    import micropip
    ...
```